### DISCUSSION TITLE
I need help with pandas

### DISCUSSION THREAD

**Discussion by Schiz0idCat** [sherlock-project/sherlock#1849] [Q&A]
When I run the program it tells me there was an error on line 12 "ModuleNotFoundError: No module named 'pandas'". I already have pandas on my pc. Help my pls.

**Comment by ausward** [comment]
If you just installed pandas using pip, and on a Windows machine, try logging out of your account and resigning in, or try restarting your computer and retrying.

**Comment by andboss25** [comment]
try doing `pip install wheel` and then `pip install pandas` OR try doing `python sherlock.py arguments` instead of `python3 sherlock.py arguments`
